launch_testing/launch_testing/__init__.py

```diff
@@ -72,3 +74,23 @@ def check(self):
         assert self.matched, \
             'Example output (%r) does not match expected output (%r)' % \
             (output_lines, self.expected_lines)
+
+def create_handler(name, ld, output_file):
```

Comment:
Either use a more descriptive argument name then `ld` or add a docblock describing it.